frontend/src/metabase/public/components/EmbedModal/SelectEmbedTypePane/SelectEmbedTypePane.tsx

```diff
@@ -72,93 +97,215 @@ export function SelectEmbedTypePane({
+        >
+          <SharingPaneButton
+            title={t`Interactive embedding`}
+            illustration={<InteractiveEmbeddingIllustration />}
+          >
+            <List>
+              {/* eslint-disable-next-line no-literal-metabase-strings -- only admin sees this */}
+              <List.Item>{t`Embed the full Metabase app with iframes.`}</List.Item>
+              <List.Item>{t`Settings to customize appearance.`}</List.Item>
+              <List.Item>{t`Includes query builder with row-level access.`}</List.Item>
+            </List>
+          </SharingPaneButton>
+        </Link>
+
+        {/* REACT SDK */}
+        <a
+          href={"https://metaba.se/sdk" + utmTags}
+          style={{ height: "100%" }}
+          target="_blank"
+          rel="noreferrer"
+        >
+          <SharingPaneButton
+            title={t`Embedded analytics SDK`}
+            badge={<BetaBadge />}
+            illustration={<SdkIllustration />}
+            externalLink
+          >
+            <List>
+              <List.Item>{t`Full control with React components.`}</List.Item>
+              {/* eslint-disable-next-line no-literal-metabase-strings -- only admin sees this */}
+              <List.Item>{t`Embed individual components of Metabase.`}</List.Item>
+              <List.Item>{t`CSS styling, user flows, usage analytics.`}</List.Item>
+              <List.Item>{t`For Pro and Enterprise plans only.`}</List.Item>
+            </List>
+          </SharingPaneButton>
+        </a>
+      </Group>
+      <Group position="apart">
+        {/* PUBLIC EMBEDDING */}
+        {isPublicSharingEnabled ? (
+          <PublicEmbedCard
+            publicEmbedCode={publicEmbedCode}
+            createPublicLink={createPublicLink}
+            deletePublicLink={deletePublicLink}
+            resourceType={resourceType}
+          />
+        ) : (
+          <Text>
+            {t`Public embeds and links are disabled.`}{" "}
+            <Link
+              variant="brand"
+              to="/admin/settings/public-sharing"
+            >{t`Settings`}</Link>
+          </Text>
+        )}
+        <a
+          className={cx(CS.link, CS.textBold)}
+          style={{ display: "flex", alignItems: "center", gap: 4 }}
+          href={
+            // eslint-disable-next-line no-unconditional-metabase-links-render -- only visible to admins
+            "https://www.metabase.com/docs/latest/embedding/introduction#comparison-of-embedding-types" +
+            utmTags
           }
-          disabled={!isPublicSharingEnabled}
-          onClick={createPublicLink}
-          data-testid="sharing-pane-public-embed-button"
-          illustration={<PublicEmbedIcon disabled={!isPublicSharingEnabled} />}
         >
-          {getPublicLinkElement()}
-        </SharingPaneButton>
+          {t`Compare options`} <Icon name="share" />
+        </a>
       </Group>
-      <InteractiveEmbeddingCTA />
     </Stack>
   );
 }
+
+const PublicEmbedCard = ({
```

Comment:
Can we split these components into different files? I really think it will improve code organization over the long-term